<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StandardFeedDetails</class>
 <widget class="QWidget" name="StandardFeedDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="MultiFeedEditCheckBox" name="m_mcbParentCategory"/>
     </item>
     <item>
      <widget class="QLabel" name="m_lblParentCategory">
       <property name="text">
        <string>Parent category</string>
       </property>
       <property name="buddy">
        <cstring>m_cmbParentCategory</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="m_cmbParentCategory">
     <property name="toolTip">
      <string>Select parent item for your feed.</string>
     </property>
     <property name="iconSize">
      <size>
       <width>12</width>
       <height>12</height>
      </size>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="MultiFeedEditCheckBox" name="m_mcbType"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Type</string>
       </property>
       <property name="buddy">
        <cstring>m_cmbType</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="m_cmbType">
       <property name="toolTip">
        <string>Select type of the standard feed.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="MultiFeedEditCheckBox" name="m_mcbEncoding"/>
     </item>
     <item>
      <widget class="QComboBox" name="m_cmbEncoding">
       <property name="toolTip">
        <string>Select encoding of the standard feed. If you are unsure about the encoding, then select &quot;UTF-8&quot; encoding.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="MultiFeedEditCheckBox" name="m_mcbTitle"/>
     </item>
     <item>
      <widget class="QLabel" name="m_lblTitle">
       <property name="text">
        <string>Title</string>
       </property>
       <property name="buddy">
        <cstring>m_txtTitle</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="LineEditWithStatus" name="m_txtTitle" native="true"/>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="MultiFeedEditCheckBox" name="m_mcbDescription"/>
     </item>
     <item>
      <widget class="QLabel" name="m_lblDescription">
       <property name="text">
        <string>Description</string>
       </property>
       <property name="buddy">
        <cstring>m_txtDescription</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <widget class="LineEditWithStatus" name="m_txtDescription" native="true"/>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="MultiFeedEditCheckBox" name="m_mcbSourceType"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Source</string>
       </property>
       <property name="buddy">
        <cstring>m_txtSource</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="m_cmbSourceType"/>
   </item>
   <item row="5" column="0">
    <widget class="MultiFeedEditCheckBox" name="m_mcbSource"/>
   </item>
   <item row="5" column="1">
    <widget class="TextEditWithStatus" name="m_txtSource" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="MultiFeedEditCheckBox" name="m_mcbPostProcessScript"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Post-processing script</string>
       </property>
       <property name="buddy">
        <cstring>m_txtPostProcessScript</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="TextEditWithStatus" name="m_txtPostProcessScript" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="HelpSpoiler" name="m_lblScriptInfo" native="true"/>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Fetch metadata</string>
     </property>
     <property name="buddy">
      <cstring>m_btnFetchMetadata</cstring>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="m_btnFetchMetadata">
       <property name="text">
        <string>Fetch it now</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LabelWithStatus" name="m_lblFetchMetadata" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="10" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="MultiFeedEditCheckBox" name="m_mcbIcon"/>
     </item>
     <item>
      <widget class="QLabel" name="m_lblIcon">
       <property name="text">
        <string>Icon</string>
       </property>
       <property name="buddy">
        <cstring>m_btnIcon</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="10" column="1">
    <widget class="QToolButton" name="m_btnIcon">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Select icon for your feed.</string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="popupMode">
      <enum>QToolButton::InstantPopup</enum>
     </property>
     <property name="autoRaise">
      <bool>false</bool>
     </property>
     <property name="arrowType">
      <enum>Qt::NoArrow</enum>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEditWithStatus</class>
   <extends>QWidget</extends>
   <header>lineeditwithstatus.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MultiFeedEditCheckBox</class>
   <extends>QCheckBox</extends>
   <header>multifeededitcheckbox.h</header>
  </customwidget>
  <customwidget>
   <class>HelpSpoiler</class>
   <extends>QWidget</extends>
   <header>helpspoiler.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LabelWithStatus</class>
   <extends>QWidget</extends>
   <header>labelwithstatus.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TextEditWithStatus</class>
   <extends>QWidget</extends>
   <header>lineeditwithstatus.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
